shared/desktop/remote/sync-avatar-props.desktop.tsx

```diff
@@ -99,12 +104,11 @@ function SyncAvatarProps(ComposedComponent: any) {
     windowParam: string
   }
 
-  class Wrapper extends React.PureComponent<WrapperProps, {usernames: Set<string>}> {
-    state = {usernames: new Set<string>()}
-    setUsernames = (usernames: Set<string>) => this.setState({usernames})
-    render() {
-      return <Connected {...this.props} usernames={this.state.usernames} setUsernames={this.setUsernames} />
```

Comment:
`setUsernames` was never being called. Looking back at the file history, I don't think using state makes sense anymore, so switching to `useRef` accomplished the same outcome.

`usernames` set is only used in `mapStateToProps` and that function should definitely not be updating state.